README.md

```diff
@@ -17,22 +17,24 @@ for providing vulnerability reports on images stored in Harbor registry as part
 ## TOC
 
 - [Requirements](#requirements)
-- [How it works?](#how-it-works)
+- [How does it work?](#how-does-it-work)
 - [Getting started](#getting-started)
   - [Prerequisites](#prerequisites)
   - [Build](#build)
-  - [Running on minikube](#running-on-minikube)
+  - [Running on Kubernetes](#running-on-kubernetes)
```

Comment:
Should this be "Running locally on Kubernetes" or "Running on Kubernetes on your laptop" to distinguish from the Kubernetes section under "Deployment"?

Reply:
It's under the Getting started section, i.e. local development to distinguish it from the Deployment section.